example/src/screens/HybridObjectTestsScreen.tsx

```diff
@@ -207,7 +207,9 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
   },
-  scrollContent: {},
+  scrollContent: {
+    paddingBottom: 65,
```

Comment:
We don't do magic values. We have `react-native-safe-area-context`, please use it.

Reply:
The change isn't because of the safe area, but because of the overlay at the bottom ("Run tests", etc.)

Reply:
yea i understand - but lets avoid magic numbers and just make it relative not absolute